src/grammar-registry.coffee

```diff
@@ -37,7 +37,7 @@ class GrammarRegistry
   #
   # Returns a {Disposable} on which `.dispose()` can be called to unsubscribe.
   onDidAddGrammar: (callback) ->
-    @emitter.on 'did-add-grammar', callback
+    @emitter?.on 'did-add-grammar', callback
```

Comment:
Why the new null checks?

Reply:
Null grammar gets constructed before registry's emitter is initialized.

https://github.com/Ingramz/first-mate/blob/d7175697aba9ba65c2da990142438946a805686d/src/grammar-registry.coffee#L15-L19